src/app-utils/constants.js

```diff
@@ -12,6 +12,21 @@ export const constants = {
     explore: function () {
       return `Explore | ${this.landing}`;
     },
+    liked: function () {
+      return `Your Likes | ${this.landing}`;
+    },
+    history: function () {
+      return `Your History | ${this.landing}`;
+    },
+    watchlater: function () {
+      return `Watch Later | ${this.landing}`;
+    },
+    login: function () {
+      return `Login | ${this.landing}`;
+    },
+    signup: function () {
+      return `Signup | ${this.landing}`;
+    },
```

Comment:
What's the use of those functions, can you explain?

Reply:
I could have directly written 'UnboxTube' in each line instead of `this.landing` , but rather than having same string again and again I used the existing landing variable which is the name of my video library and used it using `this`. So if i wish to change the library name i can change it in one place and will reflect all the other places

Reply:
Good use of "this"👍